src/Microsoft.AspNetCore.Diagnostics/DeveloperExceptionPage/Views/CompilationErrorPage.cshtml

```diff
@@ -34,6 +34,10 @@
                         <div class="titleerror">@fileName</div>
                     }
                 }
+                @if (!string.IsNullOrEmpty(errorDetail.ErrorMessage))
+                {
+                    <details>@errorDetail.ErrorMessage</details>
```

Comment:
I do not think this tag does what you think it does.

https://www.w3.org/wiki/HTML/Elements/details

> HTML Attributes
open = boolean
If present, it indicates that the details are to be shown to the user. If the attribute is absent, the details are not to be shown. [Example B] 

How about a `<div>` or `<p>`?